lib/net/http.rb

```diff
@@ -737,6 +738,11 @@ def set_debug_output(output)
     # The local port used to establish the connection.
     attr_accessor :local_port
 
+    # Whether to force a response body encoding.  If true, tries to detect
+    # the response body encoding.  If String or Encoding, uses the specified
+    # encoding.
+    attr_accessor :force_response_body_encoding
+
```

Comment:
The name `force_encoding` is intended to indicate that the method is a dirty, low-level operation. On the other hand, I think this method is highly recommended. The name should reflect the difference.

Also maybe the given String should be verified when it was set instead of the time when it calls `force_encoding`.

Reply:
I added commits to fix both of these issues.  I can update NEWS for ruby after this is merged.